.clean-install.sh

```diff
@@ -5,17 +5,19 @@
 
 PLATFORM=$1
 
-if [ "${PLATFORM}" == "" ]; then
+cd $(dirname ${0})
+
+if [[ -z "${PLATFORM}" ]] || [[ ! -a "toolchain/${PLATFORM}.config" ]]; then
   echo "Usage: $0 <platform>"
-  echo "  Where platform can be modduo[-static], modduox[-static], moddwarf or x86_64"
+  echo "  Where <platform> can be one of:"
+  echo -n "  "
+  echo $(ls toolchain | sed 's/\.config//g')
```

Comment:
coming back to this, is there a reason for echoing the output of a command, instead of running the command directly? this line does that, which is a little weird, so I am wondering if it is intentional or just an accident.